packages/concerto-vocabulary/test/vocabularymanager.js

```diff
@@ -212,6 +267,46 @@ describe('VocabularyManager', () => {
         term.should.equal('Gross Weight of the Truck');
     });
 
+    it('resolveTerms - class', () => {
+        const terms = vocabularyManager.resolveTerms(modelManager, 'org.acme', 'en-gb', 'Truck');
+        terms.Truck.should.equal('A lorry');
+    });
+
+    it('resolveTerms - class with identifier', () => {
+        const terms = vocabularyManager.resolveTerms(modelManager, 'org.acme', 'en-gb', 'Truck');
+        terms['.description'].should.equal('A heavy goods vehicle');
+    });
+
+    it('resolveTerms - property', () => {
+        const terms = vocabularyManager.resolveTerms(modelManager, 'org.acme', 'en-gb', 'Truck', 'weight');
+        terms.weight.should.equal('The weight of the truck');
+    });
+
+    it('resolveTerms - property with identifier', () => {
+        const terms = vocabularyManager.resolveTerms(modelManager, 'org.acme', 'en-gb', 'Truck', 'weight', 'description');
+        terms['.description'].should.equal('The weight of the truck in KG');
```

Comment:
This one seems a little confusing - that the term requested has to be dereferenced with a leading '.' ...

Reply:
`resolveTerms` results in this object.
```
{
  Truck: 'A truck',
  '.description': 'A vehicle capable of carrying cargo',
  '.tooltip': 'Truck',
  properties: [
    {
      weight: 'The weight of the truck',
      '.tooltip': 'Truck weight',
      '.description': 'The weight of the truck in KG'
    },
    {
      horsePower: 'The horse power of the truck',
      '.description': 'The horse power of the truck',
      '.tooltip': 'Truck HP'
    }
  ]
}
```

Reply:
What is the significance of `.` here?
Why can't we have without the `.` here? Something like this
```
locale: en
namespace: org.acme
declarations:
  - Color: A color
  - Vehicle: A road vehicle
    properties:
      - vin: Vehicle Identification Number
        tooltip: VIN
      - model: Model of the vehicle
  - Truck: A truck
    description: A vehicle capable of carrying cargo
    tooltip: Truck
    properties:
      - weight: The weight of the truck
        description: The weight of the truck in KG
        tooltip: Truck weight
      - horsePower: The horse power of the truck
        description: The horse power of the truck
        tooltip: Truck HP
```

Reply:
Okay got it.